students/ann/.gitignore

```diff
@@ -0,0 +1 @@
+.DS_Store
```

Comment:
awsome! ;)

Reply:
Could you add it to the main .gitignore?

Reply:
sure! done